docs/api/hardhat/testing.md

```diff
@@ -75,7 +57,15 @@ Local zkSync setup comes with some "rich" wallets with large amounts of ETH on b
 
 The full list of the addresses of these accounts with the corresponding private keys can be found [here](https://github.com/matter-labs/local-setup/blob/main/rich-wallets.json).
 
-Also, during the initial bootstrapping of the system, several ERC-20 contracts are deployed locally. Note, that large quantities of these ERC-20 belong to the wallet `0x36615Cf349d7F6344891B1e7CA7C72883F5dc049` (the first one in the list of the rich wallet). Right after bootstrapping the system, these ERC-20 funds are available only on L1.
+::: tip ERC-20 tokens
+
+The initial version of local node was shipped with several ERC-20 tokens deployed by default.
+
+It's no longer the case and if you need to interact with ERC-20 tokens, consider deploying them youself.
```

Comment:
A minor suggestion: "consider doing something" sounds too formal to me, but maybe that's just me

```suggestion
It's no longer the case and if you need to interact with ERC-20 tokens, you should deploy them yourself.
```

But you should definitely fix `youself` => `yourself`

Reply:
Yeah, I guess. Thanks!